docs/content/tutorials/security-monitoring.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Connect NGINX App Protect WAF to NGINX Security Monitoring"
+description: |
+  How to monitor NGINX App Protect WAF via NGINX Security Monitoring.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+---
+## Connect NGINX App Protect WAF to NGINX Security Monitoring
+
+
+This document explains how to configure NGINX Agent to send NGINX App Protect WAF metrics to NGINX Security Monitoring.
+
+## Prerequisites
+
+This guide assumes that you have an installation of NGINX Instance Manager with [NGINX Security Monitoring](https://docs.nginx.com/nginx-management-suite/installation/vm-bare-metal/install-security-monitoring/) which is reachable from the Kubernetes cluster on which F5 NGINX Ingress Controller is deployed.
+
+## Deploying NGINX Ingress Controller with GlobalConfiguration resource
+
+{{<tabs name="deploy-config-resource">}}
+
+{{%tab name="Using Helm"%}}
+
+1. Add the below arguments to the `values.yaml` file:
+    ```yaml
+    nginxAgent:
+        enable: true
+        instanceManager:
+            host: "<FQDN or IP address of NGINX Instance Manager>"
+    ```
+
+2. Follow the [Installation with Helm]({{< relref "/installation/installing-nic/installation-with-helm.md" >}}) instructions to deploy NGINX Ingress Controller with custom resources enabled, and optionally set other `nginxAgent.*` values if required.
+
+{{%/tab%}}
+
+{{%tab name="Using Manifests"%}}
+
+1. Add the below argument to the manifest file of NGINX Ingress Controller:
+
+    ```yaml
+    args:
+      - -agent=true
+      - -agent-instance-group=<NGINX Ingress Controller deployment name>
+    ```
+
+2. Create a ConfigMap with an `nginx-agent.conf` file which must be mounted to `/etc/nginx-agent/nginx-agent.conf` in the NGINX Ingress Controller pod.
+   ```yaml
+    kind: ConfigMap
+    apiVersion: v1
+    name: <configmap name>
+    namespace: <namespace where NGINX Ingress Controller will be installed>
+    data:
+      nginx-agent.conf: |-
+        log:
+          level: error
+          path: ""
+        server:
+          host: "<FQDN or IP address of NGINX Instance Manager>"
+          grpcPort: 443
+        features:
+        - registration
+        - nginx-counting
+        - metrics-sender
+        - dataplane-status
+        extensions:
+        - nginx-app-protect
+        - nap-monitoring
+        nginx_app_protect:
+          report_interval: 15s
+          precompiled_publication: true
+        nap_monitoring:
+          collector_buffer_size: 20000
+          processor_buffer_size: 20000
+          syslog_ip: 127.0.0.1
+          syslog_port: 1514
+   ```
+   See the [NGINX Agent Configuration Overview](https://docs.nginx.com/nginx-agent/configuration/configuration-overview/) for more configuration options.
+
+   *NOTE:* The `features` list must not contain `nginx-config-async` or `nginx-ssl-config` as this may cause conflicts with NGINX Ingress Controller.
```

Comment:
```suggestion
   *NOTE:* The `features` list must not contain `nginx-config-async` or `nginx-ssl-config` as this will cause conflicts with NGINX Ingress Controller.
```

Reply:
@pdabelf5 I've made a slightly different change, happy to tweak it further though!